src/assets/scripts/client/ui/UiController.js

```diff
@@ -417,6 +629,18 @@ class UiController {
         this._onClickOpenAirportDialog();
     }
 
+    /**
+     * @for UiController
+     * @method onToggleAirportGuide
+     */
+    onToggleAirportGuide() {
```

Comment:
IMHO all the show/hide dialog code should live in the respective controller. We can refactor that later though. Any thoughts?

@n8rzz @erikquinn

Reply:
I agree with you there.  The `UiController` should listen and delegate events that occur in the UI.

Looking at this a little more, that's more or less what it's doing.  with most of these options, the thing we are toggling is the active state of the footer button.  after that, we trigger a `TrackableEvent` and then trigger an `EventBus` event which then causes whatever controller is responsible for the view to respond and do it's thing from there.

This could be made more clear, for sure.  This implementation is inline with the others, for now anyway.